ext/spl/php_spl.stub.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/* ext/spl/php_spl.c */
+
+function class_implements($what, bool $autoload = true) : array|false { }
+
+function class_parents($instance, bool $autoload = true) : array|false {}
+
+function class_uses($what, bool $autoload = true) : array|false {}
+
+function spl_autoload(string $class_name, string $file_extensions = '.inc,.php') : void {}
+
+// This silently ignores non-string class names
+function spl_autoload_call($class_name) : void {}
+
+function spl_autoload_extensions(string $file_extensions = '') : string {}
```

Comment:
```suggestion
function spl_autoload_extensions(string $file_extensions = UNKNOWN) : string {}
```

Zero and two arg forms have distinct behavior.